api/app/Providers/AuthServiceProvider.php

```diff
@@ -95,7 +95,46 @@ public function resolveUserOrAbort($bearerToken, $tokenService): ?User
 
             // By this point we have verified that the token is legitimate
 
-            $userMatch = User::where('sub', $sub)->withTrashed()->first(); // 3. match "sub" claim to user 'sub' field.
+            $userMatch = User::where('sub', $sub)->withTrashed()->select(
```

Comment:
These look like the same as `User::getSelectableColumns()`? Seems like we could also use that here too.